chromadb/segment/impl/vector/local_persistent_hnsw.py

```diff
@@ -132,9 +119,6 @@ def __init__(self, system: System, segment: Segment):
         else:
             self._persist_data = PersistentData(
                 self._dimensionality,
-                self._total_elements_added,
```

Comment:
confirming this was not used anywhere else

Reply:
confirmed

Reply:
turns out this was the issue, I should have fully reasoned through it :/
`self._total_elements_added` isn't used in `local_persistent_hnsw`, but it *is* used in `local_hnsw` (the super class)